src/raygun.utilities.js

```diff
@@ -332,8 +410,76 @@ var raygunUtilityFactory = function (window) {
 
         return '';
         })(arg, url);
-      }
+      },
+      // Replace existing function on object with new, but call old one afterwards still
+      // Returns function that when called will un-enhance object
+      enhance: function(object, property, newFunction) {
+        var existingFunction = object[property];
+
+        object[property] = function enhanced() {
+          newFunction.apply(this, arguments);
+
+          if (typeof existingFunction === "function") {
+            existingFunction.apply(this, arguments);
+          }
+        };
+
+        return function unhenance() {
+          object[property] = existingFunction;
+        };
+      },
+      addEventHandler: function(element, event, handler, useCapture) {
+        var capture = useCapture || false;
+
+        if (element.addEventListener) {
+          element.addEventListener(event, handler, capture);
+        } else if (element.attachEvent) {
+          element.attachEvent('on' + event, handler);
+        }
 
+        return function() {
+          if (element.removeEventListener) {
+            element.removeEventListener(event, handler, capture);
+          } else if (element.detachEvent) {
+            element.detachEvent('on' + event, handler);
+          }
+        };
+      },
+      nodeText: function(node) {
+        var text = node.textContent || node.innerText || "";
+
+        if (["submit", "button"].indexOf(node.type) !== -1) {
+          text = node.value;
+        }
+
+        text = text.replace(/^\s+|\s+$/g, "");
```

Comment:
Maybe this could be `text.trim()`...  though this involve adding a polyfill, it could potentially be used elsewhere